src/coreclr/jit/assertionprop.cpp

```diff
@@ -4894,6 +4898,98 @@ bool Compiler::optNonNullAssertionProp_Ind(ASSERT_VALARG_TP assertions, GenTree*
     return false;
 }
 
+//------------------------------------------------------------------------
+// optAssertionIsNonHeap: see if we can prove a tree's value will be not GC-tracked
+//   based on assertions
+//
+// Arguments:
+//   op - tree to check
+//   assertions  - set of live assertions
+//   pVnBased - [out] set to true if value numbers were used
+//   pIndex - [out] the assertion used in the proof
+//
+// Returns:
```

Comment:
nit: Return Value:

Reply:
Fixed. It was a place where I copied another header in the same file that was already laid out "incorrectly" like this.